protocol/app/app.go

```diff
@@ -1167,6 +1168,24 @@ func New(
 	app.ScopedIBCKeeper = scopedIBCKeeper
 	app.ScopedTransferKeeper = scopedTransferKeeper
 
+	// Report out app version and git commit. This will be run when validators restart.
+	version := version.NewInfo()
```

Comment:
What do you think about putting this logic into a separate daemon thread that loops every 30 minutes or so?

For example, we could do the following:
- Create a goroutine to log out app version.
- Recover from all panics in goroutine.
- Log out app version immediately upon goroutine start + every 30 minutes afterwards.